Userland/Applications/FontEditor/GlyphEditorWidget.cpp

```diff
@@ -31,6 +31,9 @@ void GlyphEditorWidget::set_glyph(int glyph)
 
 void GlyphEditorWidget::paint_event(GUI::PaintEvent& event)
 {
+    if (!m_font)
+        return;
+
```

Comment:
Perhaps a better question than Lucas's is, why are we hiding GUI::Widget::font()? Per https://github.com/SerenityOS/serenity/commit/dcf64540593c3dd21e994930075f6d9315b207f6 it looks like the answer is "for consistency". But that feels strange to me. Probably a question for another time though.

Reply:
Maybe, I don't think I'll address it in this PR.